tests/components/upnp/test_config_flow.py

```diff
@@ -26,7 +30,9 @@
 
```

Comment:
We should probably have a test for

1. When the config entry doesn't have a mac address
2. When the new discovery cannot get the mac address